src/config/default.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I'm pretty sure @andrehp is advocating against having multiple config files so we'd better have only the default.

Reply:
Yes please, keeping multiple config files is a nightmare and very error prone

Reply:
done

Reply:
🙌